lib/modules/datasource/sbt-package/index.ts

```diff
@@ -189,6 +190,12 @@ export class SbtPackageDatasource extends Datasource {
           dependencyUrl,
           releases: versions.map((v) => ({ version: v })),
         };
+      } else {
+        // Try to find new releases with the maven datasource implementation
+        return await new MavenDatasource().getReleases({
+          packageName,
+          registryUrl,
+        });
```

Comment:
Looks like solution in a wrong level 🤔

Reply:
thanks, maybe you can propose the right level?

we can also retry here `lib/modules/datasource` like so
<img width="1184" alt="Screenshot 2022-07-25 at 15 09 46" src="https://user-images.githubusercontent.com/29194592/180785535-571d488b-e024-44da-9513-c7e259937682.png">

Reply:
Sure, give me some time to figure it out